package/MDAnalysis/analysis/pca.py

```diff
@@ -299,3 +300,26 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
             dot[i] = np.dot(xyz, self.p_components[:, :n_components])
 
         return dot
+
+    def cosine_content(self, pca_space, i):
+        """Measure the cosine content of the PCA projection.
+
+        Cosine content is used as a measure of convergence for a protein
+        simulation. If this function is used in a publication, please cite:
+
+        .. [BerkHess1]
```

Comment:
there is a `Reference` header in the docs you can use. Please have a look at the numpy doc examples/